build/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -e
+
+SBT_CMD="sbt ++${TRAVIS_SCALA_VERSION} validate"
```

Comment:
I'm not entirely sure what implications it might have to lock into 2.12 here, but I'm willing to try assuming if what people already doing in other projects.

Reply:
Or, perhaps, using just `release with-defaults` (drop `cross`) would allow us to keep build matrix?

Reply:
What I mean is:

```
sbt ++${TRAVIS_SCALA_VERSION} 'release with-defaults'
```

Reply:
this feels problematic but we can try it.  If `release` command triggers an updates, which bumps the version and commits the next version then i'm not entirely sure how this will effect two builds going at once.  My gut feeling says that two commits will result from this and feels a bit dangerous

Reply:
Oh I see now. That's a good point. I'm less optimistic now. Maybe an alternative would be to keep build matrix and place for PRs and "skip" the 2.11 build when we're releasing? I'd guess we can just check the string against `{TRAVIS_SCALA_VERSION}`.

Reply:
Anyways, I think we should keep it simple for now and just give up on build matrix. Let me give it a try.